migrations/versions/b1fdd85b9ca3_add_columns_to_securemessage.py

```diff
@@ -0,0 +1,26 @@
+"""add columns to securemessage
+
+Revision ID: b1fdd85b9ca3
+Revises:
+Create Date: 2018-05-24 14:46:04.172799
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b1fdd85b9ca3'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('secure_message', sa.Column('sent_datetime', sa.DateTime), schema='securemessage')
+    op.add_column('secure_message', sa.Column('read_datetime', sa.DateTime), schema='securemessage')
+
+
+def downgrade():
+    op.drop_column('secure_message', 'sent_datetime', schema='securemessage')
+    op.drop_column('secure_message', 'read_datetime', schema='securemessage')
```

Comment:
🎨 Personal preference, I always prefer using _at for datetime columns e.g. `sent_at` (and `_on` for date columns e.g. `closed_on`).

Reply:
so, `sent_at` and `read_on` for column names?  I think I only went with those names because another column had a similar name, but I was never really attached to that naming convention

Reply:
I think the sent_at and read_at sound ok, they are descriptive.